website/common/script/ops/feed.js

```diff
@@ -68,9 +70,28 @@ export default function feed (user, req = {}, analytics) {
     throw new NotAuthorized(i18n.t('messageAlreadyMount', req.language));
   }
 
+  if (amount < 0) {
```

Comment:
This looks good but I would make the check more generic with 

```
if (!Number.isInteger(amount) || amount < 0) {
    throw new BadRequest(i18n.t('invalidAmount', req.language));
}
``` 

it will protect against cases where amount is not a number